examples/cifar_demo/rise_demo_data_scientist.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connect to Clipper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# clipper_manager must be on your path:\n",
+    "import sys\n",
+    "import os\n",
+    "sys.path.append(os.path.abspath('../../management/'))\n",
+    "import clipper_manager as cm\n",
+    "# Change the username if necessary\n",
+    "user = \"ubuntu\"\n",
+    "# Set the path to the SSH key\n",
+    "key = \"\"\n",
+    "# Set the SSH host\n",
+    "host = \"\"\n",
+    "clipper = cm.Clipper(host, user, key)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Cifar\n",
+    "\n",
+    "__Set the location to the downloaded Cifar dataset:__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cifar_loc = \"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import cifar_utils\n",
+    "train_x, train_y = cifar_utils.filter_data(\n",
+    "    *cifar_utils.load_cifar(cifar_loc, cifar_filename=\"cifar_train.data\", norm=True))\n",
+    "test_x, test_y = cifar_utils.filter_data(\n",
+    "    *cifar_utils.load_cifar(cifar_loc, cifar_filename=\"cifar_test.data\", norm=True))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train Logistic Regression Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn import linear_model as lm \n",
+    "def train_sklearn_model(m, train_x, train_y):\n",
+    "    m.fit(train_x, train_y)\n",
+    "    return m\n",
+    "lr_model = train_sklearn_model(lm.LogisticRegression(), train_x, train_y)\n",
+    "print(\"Logistic Regression test score: %f\" % lr_model.score(test_x, test_y))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy Logistic Regression Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "model_added = clipper.deploy_model(\n",
+    "    \"sklearn_cifar\",\n",
+    "    1,\n",
+    "    lr_model,\n",
+    "    # TODO: change test to latest\n",
```

Comment:
These TODOs need to be addressed at the very end. `latest` is the docker build from the develop branch so it will break the demo until the PR gets merged. Holding off until we get everything else addressed.